jubatus/core/classifier/nearest_neighbor_classifier.cpp

```diff
@@ -100,12 +108,18 @@ void nearest_neighbor_classifier::train(
   }
   nearest_neighbor_engine_->set_row(id, fv);
   set_label(label);
+
+  util::concurrent::scoped_lock lk(label_mutex_);
+  labels_[label] += 1;
```

Comment:
That’s right. I try to change this patch to use ``storage::labels`` in NN-classifier.